submit/parameters.py

```diff
@@ -76,6 +78,8 @@
    fastHadd      = False                 # No fastHadd on a different T2
 nFit             = 2000                  # number of fits done in parallel
 Barrel_or_Endcap = 'ALL_PLEASE'          # Option: 'ONLY_BARREL','ONLY_ENDCAP','ALL_PLEASE'
+ContainmentCorrection = 'No' # Option: 'No', '2012reg', '2016reg','Yong'
```

Comment:
I would just put here '2012reg', since it is the one that we are currently using. 
As soon as we have the new sample and the new training we switch to 2016reg